ktxspec.adoc

```diff
@@ -926,12 +931,12 @@ to read the faces individually rather than as a block of size
 === Data Format Descriptor
 The Data Format Descriptor (_dfDescriptor_) describes the layout
 of the texel blocks in `data`.  The full specification for this is
-is Part 2, Chapters 2 to 11, of the _Khronos Data Format Specification_
-version 1.3 <<KDF13>>.
+is {url-df-spec}#pass:[_formats_and_texel_access][Part 2, Chapters 2 to
```

Comment:
What does "Part 2" refer to?

Reply:
IIRC once upon a time KDFS was divided into parts with one part for the DFD another part for the color conversion stuff and another for the block compressed formats. It doesn't make much sense now so I'll remove it.

By the way, is there a way to make Asciidoctor check external links when building the doc?

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/224/commits/d3e34e57a9154733748effcd088a58116d05cce1.